ssh/tailssh/tailssh.go

```diff
@@ -390,6 +396,13 @@ func (c *conn) ServerConfig(ctx ssh.Context) *gossh.ServerConfig {
 
 			return perms, nil
 		},
+		PublicKeyCallback: func(cm gossh.ConnMetadata, key gossh.PublicKey) (*gossh.Permissions, error) {
```

Comment:
I know you haven't found a client that initiates with a password, but add a callback for that here just in case

Reply:
Done